meshapp/src/components/svgs/logo.tsx

```diff
@@ -0,0 +1,111 @@
4184 25.2828 13.2394C25.2137 12.9721 25.1184 12.7123 25.029 12.4527C24.6978 11.4907 24.4216 10.517 24.1953 9.5185C24.1765 9.4355 24.1598 9.35193 24.1414 9.26881C24.1103 9.12814 24.155 9.383 24.156 9.3897C24.2121 9.75684 24.274 10.1212 24.3481 10.4846C24.56 11.5241 24.8165 12.5498 25.0537 13.5821C25.0633 13.6236 25.072 13.6654 25.0814 13.7069C25.0877 13.7347 25.0719 13.6507 25.066 13.6228C24.9981 13.3035 24.9222 12.9865 24.84 12.6714C24.5901 11.7135 24.3134 10.7684 24.1089 9.79633C24.0597 9.56255 23.9523 8.85691 23.9529 9.09721C23.9538 9.47303 24.0477 9.88397 24.1091 10.2475C24.2998 11.3767 24.5073 12.5051 24.7175 13.6301C24.7562 13.8374 24.7398 13.5971 24.7385 13.577C24.7227 13.3466 24.6756 13.1192 24.6331 12.8938C24.4463 11.9039 24.249 10.9164 24.0685 9.92502C24.024 9.68091 23.956 8.94724 23.9262 9.19475C23.8921 9.47766 23.964 9.84395 23.9946 10.1188C24.115 11.2013 24.3207 12.2694 24.4605 13.3487C24.4742 13.4543 24.5351 13.7641 24.497 13.6661C24.4399 13.519 24.4256 13.2558 24.3985 13.1113C24.2059 12.0866 23.9648 11.0775 23.8418 10.0374C23.8177 9.83363 23.7381 9.45052 23.7809 9.23059C23.8231 9.01437 23.8157 9.66972 23.8303 9.88954C23.8911 10.8034 23.9906 11.7129 24.0746 12.6242C24.1035 12.9376 24.1336 13.2513 24.1478 13.5662C24.1499 13.6142 24.1516 13.6623 24.1521 13.7104C24.1523 13.7232 24.1534 13.7608 24.1498 13.7487C24.143 13.7261 24.1444 13.7015 24.1423 13.6778C24.0767 12.9219 23.9879 12.1687 23.9281 11.4122C23.8837 10.8505 23.8167 10.262 23.8952 9.69837C23.9083 9.60368 23.9016 9.88947 23.9011 9.98509C23.8978 10.543 23.9128 11.0994 23.9411 11.6561C23.9697 12.2195 24.0043 12.7829 24.0434 13.3454C24.0805 13.8789 23.9454 12.2807 23.8837 11.7502C23.8016 11.0447 23.7553 10.3296 23.7512 9.6175C23.751 9.58819 23.7517 9.55888 23.7522 9.52957C23.7523 9.52106 23.7509 9.54656 23.7506 9.55507C23.7421 9.78657 23.746 10.0178 23.7491 10.2493C23.7619 11.1857 23.7588 13.9957 23.7395 13.0595C23.7242 12.314 23.6412 11.5773 23.6647 10.8296C23.6815 10.2925 23.8357 8.70046 23.8884 9.23402C23.9354 9.71067 23.885 10.2249 23.8615 10.6993C23.8313 11.308 23.7865 11.916 23.7592 12.5248C23.7483 12.7674 23.691 13.4904 23.7409 13.2531C23.8409 12.7771 23.7732 12.2265 23.7749 11.7451C23.7771 11.0794 23.7941 10.4164 23.8652 9.75411C23.9091 9.34432 23.8223 9.9995 23.8152 10.039C23.6824 10.7761 23.6211 11.5163 23.563 12.2631C23.5476 12.4607 23.5289 12.6611 23.5336 12.8596C23.5357 12.95 23.5542 12.6796 23.5576 12.5891C23.5847 11.8798 23.5642 11.161 23.7094 10.4628C23.7186 10.4186 23.8884 9.85061 23.8114 10.1955C23.7064 10.6658 23.5978 11.1276 23.5327 11.6069C23.4976 11.8654 23.4624 12.1241 23.4288 12.3828C23.3973 12.6252 23.4585 12.3492 23.4631 12.3289C23.5819 11.8069 23.6649 11.2706 23.741 10.7399C23.7689 10.5451 23.7894 10.3489 23.8198 10.1545C23.8456 9.98995 23.849 10.0637 23.8413 10.1894C23.8078 10.7325 23.7538 11.2905 23.5953 11.8108C23.5461 11.9723 23.4876 12.1356 23.4198 12.2895C23.3994 12.3358 23.414 12.1867 23.4204 12.1359C23.4563 11.8511 23.4948 11.5687 23.564 11.2905"
```

Comment:
my god, I called the discord one long... hahah Yep, think I'm going to make a task for this. This is good for now though.

Reply:
#354 related. Thank you again for creating it.